backlog_test.go

```diff
@@ -17,10 +17,14 @@ var (
 	ErrIncorrectResponse = errors.New("Response is incorrect")
 )
 
+func setup() (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
+	return setupWithPath("")
+}
+
 // setup sets up a test HTTP server along with a backlog.Client that is
 // configured to talk to that test server. Tests should register handlers on
 // mux which provide mock responses for the API method being tested.
-func setup() (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
+func setupWithPath(path string) (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
```

Comment:
I request to add comment about the method `setupWithPath(path string)` 🙏 

example:
```
// setup sets up a test HTTP server along with a backlog.Client that is
// configured to talk to that test server. Tests should register handlers on
// mux which provide mock responses for the API method being tested.
func setup() (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
	return setupWithPath("")
}

// setupWithPath sets up a test HTTP server along with a backlog.Client with the path.
func setupWithPath(path string) (client *Client, mux *http.ServeMux, serverURL string, teardown func()) {
...
```

Reply:
Fix comment.